frontend/src/components/OperationTabContainer/OperationTabContainer.tsx

```diff
@@ -68,6 +82,16 @@ const OperationTabContainer: FunctionComponent<ComponentProps> = (props) => {
             </span>
           </Alert>
 
+          <Button
+            variant="dark"
+            className="btn btn-sm btn-dark"
+            onClick={() => handleSwitchButton()}
+            style={{ position: 'absolute', right: 9, top: -20 }}
+          >
+            Switch to {location.pathname === '/queries/build/' ? 'Advanced' : value.defaultValue}{' '}
```

Comment:
Replace value.defaultValue with 'Basic'

Reply:
Oh ok so that means this works even with out context